docs/reference/ccr/getting-started.asciidoc

```diff
@@ -90,6 +90,7 @@ image::images/ccr-tutorial-clusters.png[ClusterA contains the leader index and C
 
 To configure a remote cluster from Stack Management in {kib}:
 
+. Setup a <<remote-clusters-security,secure connection>> as needed.
```

Comment:
@stefnestor this ID doesn't exist per the buildkite failed build error message

Reply:
Thanks, @leemthompo ! That'll be because the docs reorganized the pages since when I filed the PR, sorry. It looks like your team [moved the section to here](https://www.elastic.co/guide/en/elasticsearch/reference/current/remote-clusters.html#add-remote-clusters) with id reference `add-remote-clusters`. 

I'm unfamiliar with editing a branch after filing since [the guide](https://elasticco.atlassian.net/wiki/spaces/CC/pages/48203730/Filing+documentation+issues+for+Elastic+Cloud) is fully UI, but I expect it'll be easy if you'll give me a couple hours to figure it out after my next call ends :)

Reply:
Okay fixed 🙏